HomeWork/src/main/java/HomeWork6/Car.java

```diff
@@ -0,0 +1,16 @@
+package HomeWork6;
+
+public class Car {
+    private void startElectricity () {
```

Comment:
First of all, let's change the order of the methods. Public methods should be declared first and then private methods.